gradle.properties

```diff
@@ -10,10 +10,10 @@
 # Specifies the JVM arguments used for the daemon process.
 # The setting is particularly useful for tweaking memory settings.
 
-org.gradle.jvmargs=-Xmx4096m
+org.gradle.jvmargs=-Xmx1536m
 
 # use parallel execution
-org.gradle.parallel=true
+#org.gradle.parallel=true
```

Comment:
Is this intentionally part of the PR?

Reply:
Uh oh. No, It's my machine dying from Gradle. Thanks.